v4-client-js/src/clients/validator-client.ts

```diff
@@ -69,6 +69,16 @@ export class ValidatorClient {
     this._post.setSelectedGasDenom(gasDenom);
   }
 
+
+  /**
+   * @description populate account number cache in the Post module for performance.
+   */
+  populateAccountNumberCache(address: string): Promise<void> {
+    if (!this._post) throw new Error('Post module not initialized');
+
+    return this._post.populateAccountNumberCache(address);
```

Comment:
nit: it doesn't look like we use this return value so probably don't need to return?